src/app/register/register.component.ts

```diff
@@ -16,7 +16,7 @@ import {DomSanitizer, SafeHtml} from "@angular/platform-browser";
 import {transferFundsPath} from "../app-routing";
 import {WidgetInstance} from "friendly-challenge";
 import {flags} from "../featureFlags";
-import {isAllowableRedirectPath, LoginNavigationState} from "../login/login.component";
+import {isAllowableRedirectPath, LoginNavigationState, normaliseRedirectPath} from "../login/login.component";
```

Comment:
Wondering if this helper might be better in `NavigationService` or similar, so we don't have a page depending upon another?

Reply:
Done in f981ea1d